src/main/java/uk/gov/companieshouse/itemhandler/exception/RetryableErrorException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.itemhandler.exception;
+
```

Comment:
* Is this a replacement for `ServiceException`? If so, its name is better as it makes its purpose clearer. 👍  A concise Javadoc explaining its role in the contract with an underlying service will reinforce this message.

Reply:
Sure. Makes sense.